### ISSUE TITLE
Make brand_id fillable on Product model

### ISSUE THREAD

**Issue by aurelien-cd** [lunarphp/lunar#615]
The brand_id property is not fillable on Product model.

```php
protected $fillable = [
    'attribute_data',
    'product_type_id',
    'status',
];
```

This could be useful for people like me that are importing a vast amount of products from an external catalog.

**Comment by glennjacobs** [comment]
@alecritson are you happy with this proposal? Seems OK to me.

**Comment by aurelien-cd** [comment]
@alecritson I see that #864 fixes that issue on branch 0.2, maybe it could be closed now :)
